src/lang/unaccent.cpp

```diff
@@ -0,0 +1,78 @@
+#include "unaccent.hpp"
+#include <algorithm>
+
+using namespace std;
+
+/// these are our utf8 characters so far
+/// ¡ ¿ À Á È É Ê Í Ó Ú Ü  ß à á â ä  è é ê í ò ó ö  ú ü  ý Ą ą ć Č č Ę ę Ě ě Ł ł ń ň Ř ř ś Š š Ů ů ź ż ž
+/// converted through unaccent:
+/// ¡ ¿ A A E E E I O U U  ß a a a a  e e e i o o o  u u  y A a c C c E e E e L l n n R r s S s U u z z z
+/// some of them we don't have in our fonts, so hand-tuning must be done
+/// _ _ A A E E E I O U U ss a a a ae e e e i o o oe u ue y A a c C c E e E e L l n n R r s S s U u z z z
+const UnaccentTable::Item UnaccentTable::table[] = {
+    { 0xa1, 1, "!" },  // ¡
+    { 0xbf, 1, "?" },  // ¿
+    { 0xc0, 1, "A" },  // À
+    { 0xc1, 1, "A" },  // Á
+    { 0xc8, 1, "E" },  // È
+    { 0xc9, 1, "E" },  // É
+    { 0xca, 1, "E" },  // Ê
+    { 0xcd, 1, "I" },  // Í
+    { 0xd3, 1, "O" },  // Ó
+    { 0xda, 1, "U" },  // Ú
+    { 0xdc, 1, "U" },  // Ü
```

Comment:
German Ü can be written as Ue

Reply:
You are right ;) ... I've forgotten the upper case ones